templates/address-space-controller/020-ConfigMap-standard-broker-definitions.yaml

```diff
@@ -229,6 +281,14 @@ data:
         a quarter of BROKER_MEMORY_LIMIT
       name: GLOBAL_MAX_SIZE
       value: "-1"
+    - description: The name of the broker
+      displayName: AMQ Name
+      name: AMQ_NAME
+      value: broker
+    - description: Home directory to for the broker instance
+      displayName: Home
+      name: HOME
+      value: /home/artemis
```

Comment:
Remove template parameters and hardcode value in pod container env

Reply:
now removed